test/indexer/data/1b6d8348-d6e9-406a-aa6a-7ee886e52bf9.tables.tdr.json

```diff
@@ -23,7 +100,52 @@
                 {
                     "cell_line_id": "70054038-cb12-429d-bb37-69185ff1deee",
                     "version": "2019-09-20T13:43:52.455000Z",
-                    "content": "{\"describedBy\": \"https://schema.humancellatlas.org/type/biomaterial/14.5.0/cell_line\", \"schema_type\": \"biomaterial\", \"biomaterial_core\": {\"biomaterial_id\": \"Cell_line_2\", \"biomaterial_name\": \"CD45+ HLA-DR+ d9 iDCs\", \"biomaterial_description\": \"Induced Dendritic cells at reprogramming day 9\", \"ncbi_taxon_id\": [9606]}, \"type\": \"primary\", \"model_organ\": {\"text\": \"immune system\", \"ontology\": \"UBERON:0002405\", \"ontology_label\": \"immune system\"}, \"growth_conditions\": {\"growth_medium\": \"Dulbecco\\u2019s modified Eagle\\u2019s medium supplemented with 10% (v/v) FBS, 2 mM l-glutamine, and antibiotics (penicillin and streptomycin, 10 \\uf06dg/ml)\", \"culture_environment\": \"Adherent cell culture\"}, \"cell_type\": {\"text\": \"dendritic cell\", \"ontology\": \"CL:0000451\", \"ontology_label\": \"dendritic cell\"}, \"disease\": {\"text\": \"normal\", \"ontology\": \"PATO:0000461\", \"ontology_label\": \"normal\"}, \"genus_species\": [{\"text\": \"Homo sapiens\", \"ontology\": \"NCBITaxon:9606\", \"ontology_label\": \"Homo sapiens\"}], \"provenance\": {\"document_id\": \"70054038-cb12-429d-bb37-69185ff1deee\", \"submission_date\": \"2019-09-20T13:43:45.344Z\", \"update_date\": \"2019-09-20T13:43:52.455Z\", \"schema_major_version\": 14, \"schema_minor_version\": 5}}"
+                    "content": {
+                        "describedBy": "https://schema.humancellatlas.org/type/biomaterial/14.5.0/cell_line",
+                        "schema_type": "biomaterial",
+                        "biomaterial_core": {
+                            "biomaterial_id": "Cell_line_2",
+                            "biomaterial_name": "CD45+ HLA-DR+ d9 iDCs",
+                            "biomaterial_description": "Induced Dendritic cells at reprogramming day 9",
+                            "ncbi_taxon_id": [
+                                9606
+                            ]
+                        },
+                        "type": "primary",
+                        "model_organ": {
+                            "text": "immune system",
+                            "ontology": "UBERON:0002405",
+                            "ontology_label": "immune system"
+                        },
+                        "growth_conditions": {
+                            "growth_medium": "Dulbecco\u2019s modified Eagle\u2019s medium supplemented with 10% (v/v) FBS, 2 mM l-glutamine, and antibiotics (penicillin and streptomycin, 10 \uf06dg/ml)",
```

Comment:
here

Reply:
Good catch, we should triage this

Reply:
What do you mean by that? Discuss in PL?

Reply:
I was going to open a new ticket but I changed my mind and now I want to discuss during PL.